backend/services/implementations/classService.ts

```diff
@@ -99,6 +99,17 @@ class ClassService implements IClassService {
           `More than one class has the same Test Session of id ${testSessionId}`,
         );
       }
+
```

Comment:
Do we ever `getClassByTestSessionId` outside of the student login experience? i.e for data visualization? Because in that case it might make sense to have two different functions (i.e one to get the entire class and one to get students who haven't taken the test yet).

Reply:
good idea, will rename function!

Reply:
hmm options:
- `getEligibleStudentsByTestSessionId`
- `getUntestedStudentsByTestSessionId` - sounds weird
- `getTestableStudentsByTestSessionId`

Reply:
I like "testable" but not a strong preference